src/data/BPDA.ts

```diff
@@ -0,0 +1,427 @@
+export const BPDA: MapGeoJsonData = {

```

Comment:
Also - if you have an export default at the bottom, no need to use named export here. Pick either/or and this will also dictate how you import it too -

```
# ex. in /src/foo.ts 
export const foo = {foo: 'bar'}

# then you should use this notation when importing:
import { foo } from 'src/foo';
```

if you use export default, it implies there is only one export from that entire file
```
# ex. in /src/foo.ts
const foo = {foo: 'bar'}

export default foo

# then in the other file which you import
import foo from 'src/foo';
```

IMO it comes down to a stylistic preference, I don't think there's any performance or other perceived benefit one vs the other. Personally I like named export because it gives slightly more flexibility in that you can have multiple exports in one file -

```
# in /src/foo.ts
export const foo = {foo: 'bar'}
export const bar = {bar: 'baz'}
export const boop = {boop: 'doot'}

# now you can reference one or all three of these if you wish in the file you are importing them
import { foo, bar, boop } from 'src/foo';
```